<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintXClass</class>
 <widget class="QMainWindow" name="PaintXClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>883</width>
      <height>802</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="sketchpad" native="true">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>800</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="foreColorBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backColorBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSlider" name="penWidthSlider">
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="penWidthSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Layers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="paintObjectList"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionArrow"/>
   <addaction name="actionPenceil"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionRoundRectangle"/>
   <addaction name="actionCircle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionPolgon"/>
   <addaction name="actionClear"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionArrow">
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/arrow.png</normaloff>:/PaintX/res/arrow.png</iconset>
   </property>
   <property name="text">
    <string>箭头</string>
   </property>
   <property name="toolTip">
    <string>箭头</string>
   </property>
  </action>
  <action name="actionPenceil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/penceil.png</normaloff>:/PaintX/res/penceil.png</iconset>
   </property>
   <property name="text">
    <string>Penceil</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/line.png</normaloff>:/PaintX/res/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/rectangle.png</normaloff>:/PaintX/res/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionRoundRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/roundrectangle.png</normaloff>:/PaintX/res/roundrectangle.png</iconset>
   </property>
   <property name="text">
    <string>RoundRectangle</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/circle.png</normaloff>:/PaintX/res/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/ellipse.png</normaloff>:/PaintX/res/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionPolgon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/多边形.png</normaloff>:/PaintX/res/多边形.png</iconset>
   </property>
   <property name="text">
    <string>Polgon</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/clear.png</normaloff>:/PaintX/res/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="PaintX.qrc">
     <normaloff>:/PaintX/res/export.png</normaloff>:/PaintX/res/export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PaintX.qrc"/>
 </resources>
 <connections/>
</ui>
